### ISSUE TITLE
`valid-title`: false positives when using `it.each`

### ISSUE THREAD

**Issue by swissspidy** [jest-community/eslint-plugin-jest#458]
With version 23, if I'm using something like this:

```js

describe( 'myFunction', () => {
	it.each( [
		[ 123, true ],
		[ 456, true ],
		[ 789, false ],
		[ 0, false ],
	] )( 'should do something',
		( number, myBool ) => {
			expect( muFunction( number ) ).toBe( myBool );
		},
	);
} );
```

The `jest/valid-title` complains about the array not being a valid title.

I see that there have been a few changes to `valid-title` in the last releases. So perhaps an issue like #334 is now appearing again, or at least this one here is very similar.

`src/rules/__tests__/valid-title.test.ts` doesn't have any test cases for `it.each` (except a single one from #436). Adding some would be good I think.

**Comment by G-Rath** [comment]
Hmm interesting - that's b/c of `valid-describe`.

I'll get a fix out shortly.

**Comment by swissspidy** [comment]
Thanks @G-Rath, I appreciate your quick response!

**Comment by MikeYermolayev** [comment]
Hm, looks like it's still an issue in "eslint-plugin-jest": "24.1.2",

![image](https://user-images.githubusercontent.com/8783498/99556184-43112480-29d2-11eb-9074-42ecd2db762c.png)

@G-Rath  Could you please take a look?

**Comment by MikeYermolayev** [comment]
Oops. I'm so sorry. I used a wrong syntax and eslint error is expected here
